RealmSwift-swift2.2/Tests/TestUtils.mm

```diff
@@ -53,3 +53,25 @@ void RLMAssertThrows(XCTestCase *self, dispatch_block_t block, NSString *name, N
 bool RLMHasCachedRealmForPath(NSString *path) {
     return RLMGetAnyCachedRealmForPath(path.UTF8String);
 }
+
+void RLMAssertThrowsWithReasonMatchingSwift(XCTestCase *self, dispatch_block_t block, NSString *regexString, NSString *message, NSString *fileName, NSUInteger lineNumber) {
```

Comment:
This code is duplicated from from the Realm Objective-C `TestUtils.mm` file. Would it be preferable to include that file in both test targets? We already have precedent of duplicating `RLMAssertThrows` across multiple files, but it definitely is gross having to maintain both separately.